eth/tokens.json

```diff
@@ -1506,6 +1506,15 @@
   "forum": "https://forum.saturn.network/t/elementor-coin-project/4726"
   },
   {
+  "name": "Civic",
+  "address": "0x9fe1125896af883c584740805429d16bf615132b",
+  "symbol": "CVC",
+  "decimals": 8,
+  "logo": "https://github.com/Micpittser84/assets/blob/master/eth/logo/CVC.png",
+  "website": "https://www.civic.com",
+  "forum": "https://forum.saturn.network/t/elementor-coin-project/4726"
```

Comment:
Pretty sure CVC is Civic, not Elementor Coin. Please find or create the correct forum topic.